components/scream/src/physics/cld_fraction/cld_fraction_main_impl.hpp

```diff
@@ -47,15 +47,15 @@ ::calc_icefrac(
   const MemberType& team,
   const Int& nk,
   const uview_1d<const Spack>& qi,
-  const uview_1d<Spack>&       aist)
+  const uview_1d<Spack>&       ice_cld_frac)
 {
   team.team_barrier();
   const Int nk_pack = ekat::npack<Spack>(nk);
   Kokkos::parallel_for(
     Kokkos::TeamThreadRange(team, nk_pack), [&] (Int k) {
       auto icecld = qi(k) > 1e-5;
-      aist(k) = 0.0;
-      aist(k).set(icecld, 1.0);
+      ice_cld_frac(k) = 0.0;
+      ice_cld_frac(k).set(icecld, 1.0);
```

Comment:
Note to self: we should add a method for Pack like `set(Mask m, Scalar val_if_m_true, Scalar val_if_m_false)`. This would allow to merge the two lines above into one single for loop.